collections/luci/Makefile

```diff
@@ -9,7 +9,8 @@ include $(TOPDIR)/rules.mk
 LUCI_TYPE:=col
 LUCI_BASENAME:=luci
 
-LUCI_TITLE:=Standard OpenWrt set including full admin with ppp support and the default Bootstrap theme
+LUCI_TITLE:=LuCi interface with Uhttpd as Webserver
```

Comment:
LuCI

The explanation should also tell the casual user clearly that this is the default variant.
So I suggest:
"LuCI interface with Uhttpd as webserver (default)"